packages/web-components/src/divider/divider.template.ts

```diff
@@ -0,0 +1,14 @@
+import { ElementViewTemplate, html } from '@microsoft/fast-element';
+import type { Divider } from './divider.js';
+
+/**
+ * Template for the Divider component
+ * @public
+ */
+export const template: ElementViewTemplate<Divider> = html<Divider>`
+  <template role="${x => x.role}" aria-orientation="${x => x.orientation}">
+    <span>
```

Comment:
Do we need this element?

Reply:
Ryan and I are working through this.

Reply:
Yes, I do need the span. I have to apply pseudo elements for the divider lines. I need them on this external span; that way, if the slot is empty the lines still show up.

Reply:
Could this not target the host element though instead of the span?

Reply:
Fixed. Thanks for the suggestion!